fjord/feedback/static/js/android_feedback.js

```diff
@@ -0,0 +1,43 @@
+(function($) {
+  'use strict';
+
+  var url = decodeURIComponent(window.location.href);
+  var browserReviewUrl = 'https://play.google.com/store/apps/details?id=org.mozilla.CHANNEL#details-reviews';
+  var deviceReviewUrl = 'market://details?id=org.mozilla.CHANNEL';
+  var reviewUrl = '';
+
+  /* decide whether they're on Fx Android or not */
+  var fromDevice = /source=feedback-prompt/.test(url);
```

Comment:
I filed a bug to make sure we have the params this script expects: https://bugzilla.mozilla.org/show_bug.cgi?id=1239513